tui/actions.go

```diff
@@ -112,30 +113,56 @@ func (ui *UI) AnalyzePath(path string, parentDir *analyze.Dir) error {
 	return nil
 }
 
-func (ui *UI) deleteSelected() {
+func (ui *UI) deleteSelected(isEmpty bool) {
```

Comment:
`isEmpty` sounds like a flag indicating that the target directory is empty. Maybe `shouldEmpty` would be better? What do you think?

Reply:
Yes, `shouldEmpty` is much better.